src/service.ts

```diff
@@ -165,10 +165,10 @@ export class DebugService implements IDebugger {
   }
 
   /**
-   * Whether the current thread is stopped.
+   * Whether there exist a thread in stopped stated.
    */
-  isThreadStopped(): boolean {
-    return this._stoppedThreads.has(this.currentThread());
+  hasThreadStopped(): boolean {
```

Comment:
Or maybe `hasStoppedThread()`?

Reply:
yep that would be consistent with the name of the variable